--- ./src/validate.c.orig	2019-08-01 20:53:16.583338594 +0300
+++ ./src/validate.c		2019-08-01 20:53:44.573646209 +0300
@@ -377,7 +377,7 @@ static DesktopKeyDefinition registered_action_keys[] = {
 
 /* This should be the same list as in xdg-specs/menu/menu-spec.xml */
 static const char *show_in_registered[] = {
-    "GNOME", "GNOME-Classic", "GNOME-Flashback", "KDE", "LXDE", "LXQt", "MATE", "Razor", "ROX", "TDE", "Unity", "XFCE", "EDE", "Cinnamon", "Pantheon", "Budgie", "Enlightenment", "Deepin", "Old"
+    "GNOME", "GNOME-Classic", "GNOME-Flashback", "KDE", "LXDE", "LXQt", "MATE", "Razor", "ROX", "TDE", "Unity", "XFCE", "EDE", "Cinnamon", "Pantheon", "Budgie", "Enlightenment", "Deepin", "Old", "OPENBOX"
 };
 
 static struct {
@@ -522,6 +522,7 @@ static struct {
   { "Adult",                  FALSE, FALSE, FALSE, { NULL }, { NULL } },
   { "Core",                   FALSE, FALSE, FALSE, { NULL }, { NULL } },
   { "KDE",                    FALSE, FALSE, FALSE, { NULL }, { "Qt", NULL } },
+  { "LXQt",                   FALSE, FALSE, FALSE, { NULL }, { "Qt", NULL } },
   { "GNOME",                  FALSE, FALSE, FALSE, { NULL }, { "GTK", NULL } },
   { "XFCE",                   FALSE, FALSE, FALSE, { NULL }, { "GTK", NULL } },
   { "GTK",                    FALSE, FALSE, FALSE, { NULL }, { NULL } },
