diff -uraN desktop-file-utils-0.27/src/validate.c omv-desktop-file-utils-0.27/src/validate.c
--- desktop-file-utils-0.27/src/validate.c	2023-10-05 17:07:13.000000000 +0200
+++ omv-desktop-file-utils-0.27/src/validate.c	2023-12-17 17:07:41.771065410 +0100
@@ -380,7 +380,7 @@
 
 /* This should be the same list as in xdg-specs/menu/menu-spec.xml */
 static const char *show_in_registered[] = {
-    "GNOME", "GNOME-Classic", "GNOME-Flashback", "KDE", "LXDE", "LXQt", "MATE", "Razor", "ROX", "TDE", "Unity", "XFCE", "EDE", "Cinnamon", "Pantheon", "Budgie", "Enlightenment", "DDE", "Endless", "Old"
+    "GNOME", "GNOME-Classic", "GNOME-Flashback", "KDE", "LXDE", "LXQt", "MATE", "Razor", "ROX", "TDE", "Unity", "XFCE", "EDE", "Cinnamon", "Pantheon", "Budgie", "Enlightenment", "DDE", "Endless", "Old", "OPENBOX"
 };
 
 static struct {
