diff -Naur desktop-file-utils-0.22/src/validate.c desktop-file-utils-0.22.tpg/src/validate.c
--- desktop-file-utils-0.22/src/validate.c	2013-07-19 07:19:42.000000000 +0000
+++ desktop-file-utils-0.22.tpg/src/validate.c	2015-03-01 23:17:47.044777440 +0000
@@ -363,7 +363,7 @@
 };
 
 static const char *show_in_registered[] = {
-  "GNOME", "KDE", "LXDE", "MATE", "Razor", "ROX", "TDE", "Unity", "XFCE", "Old"
+  "GNOME", "KDE", "LXDE", "MATE", "Razor", "ROX", "TDE", "Unity", "XFCE", "Cinnamon", "EDE", "LXQt", "Old"
 };
 
 static struct {
